packages/sanity/src/studio/renderStudio.tsx

```diff
@@ -1,5 +1,5 @@
-import React from 'react'
-import {render} from 'react-dom'
+import React, {StrictMode} from 'react'
+import {createRoot} from 'react-dom/client'
```

Comment:
`createRoot` enables some of the new rendering features in React. But to use concurrent rendering we'll have to use new APIs like `startTransition`. The scope of this PR is to lay the groundwork and allow us to gradually adopt the new features in future releases.